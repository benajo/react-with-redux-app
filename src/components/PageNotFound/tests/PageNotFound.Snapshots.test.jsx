import React from 'react';
import renderer from 'react-test-renderer';
import PageNotFound from '../PageNotFound';

it('render PageNotFound component', () => {
  const tree = renderer.create(<PageNotFound />);
  expect(tree).toMatchSnapshot();
});
