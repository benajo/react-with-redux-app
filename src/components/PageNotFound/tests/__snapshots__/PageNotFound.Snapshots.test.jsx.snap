// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render PageNotFound component 1`] = `
<h1>
  Oops! Page not found.
</h1>
`;
